src/networking.c

```diff
@@ -3286,14 +3286,13 @@ sds catClientInfoString(sds s, client *client) {
         replBufBlock *cur = listNodeValue(client->ref_repl_buf_node);
         used_blocks_of_repl_buf = last->id - cur->id + 1;
     }
-
```

Comment:
If there are any changes later, please add this line back, otherwise we can just leave it alone.